var/spack/repos/builtin/packages/py-Genshi/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import version, extends, depends_on
+from spack import Package
+
+
+class PyGenshi(Package):
```

Comment:
can this one be lowercased to `py-genshi` for the package name?  Basically that just requires renaming the directory.